client/src/css/Ellie.css

```diff
@@ -0,0 +1,62 @@
+@import url('https://fonts.googleapis.com/css2?family=Cabin&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Comfortaa:wght@500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+.welcome {
+  margin: 5%;
+  align-items: center;
+  justify-content: left;
+}
+
+#home-picture {
+  width: 200px;
+  height: 200x;
+}
+
+.web-picture {
+  width: 200px;
+  height: 200px;
+  padding-left: 5%;
+}
+
+.picture {
+  order: 1;
+}
+
+.text {
+  order: 2;
+  font-size: 20px;
+  font-family: 'Poppins', sans-serif;
+  text-align: left;
+}
+
+.abme-text {
+  padding-top: 70px;
+  font-family: 'Poppins', sans-serif;
+  font-size: small;
+  text-align: left;
+}
+
+a:link {
+  color: black;
+  background-color: transparent;
+  text-decoration: none;
+}
+
+a:hover {
+  color: rgb(125, 168, 138);
+  background-color: transparent;
+  text-shadow: black;
+}
```

Comment:
Use classNames for these as well. General tags will affect all links/hovers in the entire repo, so your styles may affect other people's pages.